### PR TITLE
OCPBUGS-45726 [IBMCloud] Require additional service endpoint overrides

### PR DESCRIPTION
**Pull request by MayXuQQ**
verify 4.17 bug [OCPBUGS-45726](https://issues.redhat.com/browse/OCPBUGS-45726)

### CODE REVIEW

**Comment by MayXuQQ** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.17-installer-rehearse-ibmcloud-dis

File: ci-operator/step-registry/cucushift/installer/check/ibmcloud/custom-endpoints/cucushift-installer-check-ibmcloud-custom-endpoints-commands.sh [hunk 1]
```diff
@@ -88,8 +88,8 @@ fi
 export KUBECONFIG=${SHARED_DIR}/kubeconfig
 
 isPre418="True"
-isPreVersion "4.18" || isPre418="False"
-echo "is Pre 4.18 version: ${isPre418}"
+isPreVersion "4.17" || isPre418="False"
+echo "is Pre 4.17 version: ${isPre418}"
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/check/ibmcloud/custom-endpoints/cucushift-installer-check-ibmcloud-custom-endpoints-commands.sh:92]:
```suggestion
isPre417="True"
isPreVersion "4.17" || isPre417="False"
echo "is Pre 4.17 version: ${isPre417}"
```

**Comment by MayXuQQ** [comment]
@jianlinliu  can I get an approved label? thanks a lot

**Comment by jianlinliu** [comment]
cc @liangxia for approval

**Pull request merged**